content/actions/managing-workflow-runs/manually-running-a-workflow.md

```diff
@@ -32,3 +32,11 @@ To trigger the `workflow_dispatch` event on {% data variables.product.prodname_d
 When using the REST API, you configure the `inputs` and `ref` as request body parameters. If the inputs are omitted, the default values defined in the workflow file are used. 
 
 For more information about using the REST API, see the "[Create a workflow dispatch event](/rest/reference/actions/#create-a-workflow-dispatch-event)."
+
+### Example YAML 
```

Comment:
I agree that folks landing on this page might want to know about both the workflow configuration and how to use the UI, but we do already have an example workflow for this case on another page. Both sections "Running a workflow using the REST API" and "Running a workflow on GitHub" require the workflow to be configured for the `workflow_dispatch`, which is why we've included the paragraph linking to the events at the top of the article
https://github.com/github/docs/pull/358/files#diff-0d421d891ef5301f3a44801c5f65c183d09e5ffb890dfc8c95ea68f29311a402R13

But I realize that the link we include up there doesn't directly link to the `workflow_dispatch` event. 😬 It should.

To address the visibility of that link, maybe we can create a new section header for the information linking to the `workflow_dispatch` event description. So you could replace the paragraph at the top with:

```
### Configuring a workflow to run manually

To run a workflow manually, the workflow must be configured to run on the `workflow_dispatch` event. For more information about configuring the `workflow_dispatch` event, see "[Events that trigger workflows](/actions/reference/events-that-trigger-workflows#workflow_dispatch)."
```

We could then also include your example here as an alternate example on this page: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#example-workflow-configuration

For example:


#### Example 
```yaml
on: workflow_dispatch
```

#### Example configuration with inputs

This example defines the name and home inputs and prints them using the github.event.inputs.name and github.event.inputs.home contexts. If a name isn't provided, the default value 'Mona the Octocat' is printed.

```yaml
name: Manually triggered workflow
on:
  workflow_dispatch:
    inputs:
      name:
        description: 'Person to greet'
        required: true
        default: 'Mona the Octocat'
      home:
        description: 'location'
        required: false

jobs:
  say_hello:
    runs-on: ubuntu-latest
    steps:
    - run: |
        echo "Hello ${{ github.event.inputs.name }}!"
        echo "- in ${{ github.event.inputs.home }}!"
```

Reply:
This makes sense. Would you like me to do that work in this PR, create a new one, or would it be something y'all would want to do yourselves?

Reply:
@bnb feel free to update this PR if you'd like to. Our team probably won't be able to get to this for a bit, so any contributions you make here would be super helpful!